### PR TITLE
Improve update check

### PR DESCRIPTION
**Pull request by Gared**
This change will change the user-agent for the requests to get the last version information from "axios/1.5.0" to "Etherpad/1.9.3".
I also added a caching for one hour to reduce those calls.

### CODE REVIEW

File: src/node/utils/UpdateCheck.js [hunk 1]
```diff
@@ -2,23 +2,35 @@
...
-              return
-            }
-            return await Promise.resolve(infos);
-          }
-          catch (err) {
-            return   await Promise.reject(err);
-          }
-        })
+  return axios.get('https://static.etherpad.org/info.json', {headers: headers})
+  .then(async resp => {
```
Comment (SamTV12345) [inline src/node/utils/UpdateCheck.js:19]: axios has a separate method for handling error. Everything that is not 2xx like 404, 500, etc. are handled in .catch after .then. There you can put the error handling measures.

Comment (Gared) [reply src/node/utils/UpdateCheck.js:19]: I reformatted the code and have not changed the exception handling, but you're right. I improved the exception handling. Please have a look again 🙂

Thanks, but don't expect too much. Just wanted to improve a little bit how my server is getting called 😅

Comment (SamTV12345) [reply src/node/utils/UpdateCheck.js:19]: Nice. I'll take a look asap and check if it's working. 

Ah okay. Thought I might get somebody back to the project 😂.

**Review by SamTV12345 — APPROVED** [review]
LGTM and works without a problem.

**Pull request merged**